plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1981,6 +1985,11 @@ def onResetScribbles(self):
         # remove "scribbles" segments from label
         self.onClearScribblesSegmentNodes()
 
+        # re-add ROI node if removed
+        if self.ui.scribblesPlaceWidget.currentNode() is None:
+            scribblesROINode = self.createScribblesROINode()
```

Comment:
This is not the right place for re-initialising - please do it in the `onStartScribbling()` function or create a separate `initialiseROINodes()` function that gets called everytime a new sample is loaded...